.gcloudignore

```diff
@@ -20,10 +20,10 @@
 # Ignored by the build system
 /setup.cfg
 
-**/node_modules/**
+static/node_modules/
 
-.env/
+server/dist/
 
-screenshots/*.png
+.env/
```

Comment:
Should we use `**/.env/` to include the venvs from the tools directory?

Reply:
Done